FMI-CROSS-CHECK-RULES.md

```diff
@@ -131,14 +103,16 @@ Vendors are encouraged to produce FMUs that are also free of all FMI Compliance
 
 FMUs for the “c-code” platform are exempt from this rule until the FMI Compliance Checker supports the “c-code” platform.
 
+// TODO: what about FMPy here? Second alternative? Drop this entirely?
```

Comment:
It would be great to include source code FMUs to the XC.  Perhaps only for FMI3.0 FMUs where we have the build configuratio?

Reply:
c-code s already a platform and there are already c-code FMUs and import results for those.